.antora/antora.yml

```diff
@@ -21,7 +21,7 @@ asciidoc: # Sets global AsciiDoc attributes that are applied for every page of t
     # START - Mandatory ASAM attributes
     use-antora-rules: true
     THIS_STANDARD: ASAM OpenMATERIAL<sup>®</sup> # Normal attribute
-    THIS_STANDARD_TITLE: 'ASAM OSI^®^' # Normal attribute
+    THIS_STANDARD_TITLE: 'ASAM OpenMATERIAL^®^' # Normal attribute
```

Comment:
This has nothing to do with this PR, please revert and fix on a different branch @FKlopfer.

Reply:
Revert changes. Will fix it on another branch.